x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/node_details.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { NODE_WIDTH } from '../constants';
+import { Tag } from './tag/tag';
+import { Ips } from './ips/ips';
+import { CountryFlags } from './country_flags/country_flags';
+import { Label } from './label/label';
+import { NODE_TAG_WIDTH } from './styles';
+
+interface NodeDetailsProps {
+  count?: number;
+  tag?: string;
+  label?: string;
+  ips?: string[];
+  countryCodes?: string[];
+}
+
+export const NodeDetails = ({ count, tag, label, ips, countryCodes }: NodeDetailsProps) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <EuiFlexGroup
+      direction="column"
+      gutterSize="s"
+      css={css`
+        width: ${NODE_TAG_WIDTH}px;
+        margin-left: ${-(NODE_TAG_WIDTH - NODE_WIDTH) / 2}px;
+      `}
+    >
+      <EuiFlexItem grow={false}>
+        <Tag count={count} text={tag} />
+      </EuiFlexItem>
+      <EuiFlexItem grow={false} css={{ alignItems: 'center', gap: euiTheme.size.xxs }}>
+        <Label text={label} />
+        <Ips ips={ips} />
+        {countryCodes && <CountryFlags countryCodes={countryCodes} />}
```

Comment:
<img width="500" alt="Screenshot 2025-07-27 at 14 22 01" src="https://github.com/user-attachments/assets/03564113-a45c-4c06-aede-3bb55fbae516" />

Pay attention when country codes is empty the component is being rendered